src/main/scala/org/scalatestplus/selenium/WebBrowser.scala

```diff
@@ -1577,7 +1575,7 @@ trait WebBrowser {
         case e: org.openqa.selenium.NoSuchFrameException => 
           throw new TestFailedException(
                      (_: StackDepthException) => Some("Frame at index '" + index + "' not found."),
-                     None,
+                     Option(e),
```

Comment:
I'm not sure the intent here was to ignore the cause, so I'm adding it here and at some places below.